src/components/TrackingConsent.vue

```diff
@@ -0,0 +1,309 @@
+<template>
+    <div
+        id="tracking-consent"
+        class="tracking-consent-banner nq-shadow"
+        :class="theme"
+        v-if="!consentKnown"
```

Comment:
In some cases we want to have additional control over whether the ui should be displayed or not.
For example, when the multi currency checkout disclaimer is visible as overlay, the tracking consent overlay should not be rendered.
Can you add a `allowUi` prop for that purpose?

Reply:
yeah sure